test/createAzureResource.test.ts

```diff
@@ -120,6 +121,16 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.state, 'Running', `Function App state should be 'Running' rather than ${createdApp.state}.`);
     });
 
+    test('Add new setting', async () => {
+        const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
+        assert.ok(createdApp, `Function App ${resourceName1} not found`);
+        ext.ui = new TestUserInput([resourceName1, applicationSettings[0], applicationSettings[1]]);
+        await vscode.commands.executeCommand('azureFunctions.appSettings.add');
+        const listapplicationSettings: WebSiteManagementModels.StringDictionary = await webSiteClient.webApps.listApplicationSettings(resourceName1, resourceName1);
+        const applicationSettingsValue: string | undefined = applicationsettings(listapplicationSettings.properties, applicationSettings[0]);
```

Comment:
This is more complicated than what you need to do.  You should be able to call `listApplicationSettings` and then check the `.properties` object on the return value.  `properties` contains an object where the key is the setting name name and the value is the setting value.

Here is the structure of the returned object: 
![image](https://user-images.githubusercontent.com/5290572/61091658-d5c1ac80-a3f7-11e9-96da-2337c5dd94aa.png)